src/platform/user/profile/vet360/containers/VAPProfileField.jsx

```diff
@@ -288,9 +298,37 @@ class VAPProfileField extends React.Component {
     return (
       <div className="vet360-profile-field" data-field-name={fieldName}>
         <Modal
-          title={`You’re currently editing your ${VET360.FIELD_TITLES[
-            activeEditView
-          ]?.toLowerCase()}`}
+          title={'Are you sure?'}
+          status="warning"
+          visible={this.state.showConfirmCancelModal}
+          onClose={() => {
+            this.setState({ showConfirmCancelModal: false });
+          }}
+        >
+          <p>
+            {' '}
```

Comment:
is this needed?

Reply:
ESLint added in that awkward white space, I cannot save the file when I take it away :(

Reply:
It's a `prettier` thing :shrug: